src/graph/backend/dnnl/kernels/sdp.hpp

```diff
@@ -65,17 +68,25 @@ struct sdp_base_t : public kernel_base_t {
         if (enable_ukernel) {
             kernel = std::make_shared<sdp_primitive_kernel_t<quantized>>();
             ret = kernel->compile_impl(part, g_engine, inputs, outputs);
+            if (ret == status::success)
+                VDISPATCH_GRAPH_SDP("dispatch to sdp_primitive_kernel");
```

Comment:
```suggestion
                VDISPATCH_GRAPH_SDP("dispatch to sdp_primitive_kernel_t");
```
Just in case you intention is to show the class name.

Reply:
fixed. thanks